Library/Homebrew/cmd/help.rb

```diff
@@ -32,14 +32,34 @@
 # NOTE The reason the string is at the top is so 25 lines is easy to measure!
 
 module Homebrew
-  def help
-    puts HOMEBREW_HELP
-  end
+  def help(cmd = nil, empty_argv = false)
```

Comment:
This will make the call site look `Homebrew.help cmd, :empty_argv => empty_argv`. If this doesn't bother you, I'm happy to make the change according to your suggestion.